docs/user-guide/getting-started-with-docker-and-lets-encrypt.md

```diff
@@ -0,0 +1,181 @@
+# Docker & Traefik
+In this use case, we want to use Traefik as a *layer-7* load balancer with SSL termination for a set of microservices used to run a webapplication. We also want to automatically *discover any services* on the Docker host and let Traefik reconfigure itself automatically when containers get created (or shut down) so HTTP traffic can be routed accordingly. In addition, we want to use Let's Encrypt to automatically generate and renew SSL certificates per hostname.
+
+## Setting up
+In order for this to work, you'll need a server with a public IP address, with Docker installed on it. In this example, we're using the fictitious domain *my-awesome-app.org". In real-life, you'll want to use your own domain and have the DNS configured accordingly so the hostname records you'll want to use point to the aforementioned public IP address.
+
+## Networking
+Docker containers can only communicate with each other over TCP when they share at least one network. This makes sense from a topological point of view in the context of networking, since Docker under the hood creates IPTable rules so containers can't reach other containers *unless you'd want to*. In this example, we're going to use a single network called `web` where all containers that are handling HTTP traffic (including Traefik) will reside in.
+
+On the Docker host, run the following command:
+
+`$ docker network create web`
+
+Now, let's create a directory on the server where we will configure the rest of Traefik:
+
+`$ mkdir -p /opt/traefik`
+
+Within this directory, we're going to create 3 empty files:
+
+```sh
+$ touch /opt/traefik/docker-compose.yml
+$ touch /opt/traefik/acme.json
```

Comment:
The acme.json file must only be accessed in read/wrie by the owner that why it should be better to set these commands : 
`touch /opt/traefik/acme.json  && chmod 600 /opt/traefik/acme.json`